app/custom-oauth/server/oauth_helpers.js

```diff
@@ -0,0 +1,33 @@
+import { addUserRoles, removeUserFromRoles } from '../../authorization';
+
+/**
+*/
+export function mapRolesFromSSO(user, identity, roleClaimName) {
+	if (user && roleClaimName) {
+		// Adding roles
+		if (identity[roleClaimName]) {
```

Comment:
Do you have documentation on what format these are usually in?  Would be great to see some examples of this being used.  Is always array?  Or is it space separated ever?

Reply:
I see okta allows setting groups scope and then returning as `group` claim in the jwt.  

https://developer.okta.com/docs/how-to/creating-token-with-groups-claim/#step-five-decode-the-jwt-to-verify

Do you have any other example?

Reply:
it's usally an array, see [here](https://developer.okta.com/docs/how-to/creating-token-with-groups-claim/#step-five-decode-the-jwt-to-verify)

Reply:
maybe in addition to this lets verify its an array, if not for now just skip it?

Reply:
sure, i will add this